src/interfaces/filing-interfaces/filing-interfaces.ts

```diff
@@ -185,7 +185,11 @@ export interface RestorationFilingIF {
     nameTranslations?: NameTranslationIF[]
     nameRequest?: RegistrationNameRequestIF
     parties: PartyIF[]
-    offices: IncorporationAddressIF
-    contactPoint: ContactPointIF
+    offices: RegisteredRecordsAddressesIF | object
+    contactPoint: {
+      email: string
+      phone: string
+      extension?: number
+    }
```

Comment:
I think you should restore the previous type for contact point.

Reply:
okay